SPEC/SWARM.md

```diff
@@ -1,6 +1,7 @@
 # Swarm API
 
 * [swarm.addrs](#swarmaddrs)
+* [swarm.localAddrs](#swarmlocaladdrs)
```

Comment:
Please can this be arranged alphabetically? i.e. `swarm.localAddrs` should be after `swarm.disconnect` and before `swarm.peers`.

Reply:
Items in this list are already ordered logically than alphabetically. Should I fix them up too?

Reply:
Yes please